internal/json/patch/edittree.go

```diff
@@ -1,11 +1,12 @@
 package patch
 
 import (
-	"encoding/json"
 	"fmt"
+	"strconv"
 	"strings"
 
 	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/internal/bitvector"
```

Comment:
I think the file might be missing? 🔍

Reply:
Ack! I did forget to add that dependency. :facepalm: I have vendored in one of the data structures from [Dropbox's Go utilities library](https://github.com/dropbox/godropbox), specifically the [bit-vector container](https://github.com/dropbox/godropbox/tree/master/container/bitvector).

The bit-vectors make some of the bookkeeping less awful when tracking edits to Arrays, and the implementation is straightforward. If higher efficiency/speed is needed there in the future, there are other bit-vector libraries out there that could easily replace the vendored one.